config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-1.yaml

```
@@ -0,0 +1,300 @@
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    optional: true
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.0$
+    path_alias: sigs.k8s.io/cluster-api
+    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|internal|scripts|test|third_party|util|webhooks|version)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220121-bb77a9cb93-1.23
+        args:
+        - runner.sh
+        - "./scripts/ci-e2e.sh"
+        env:
+        - name: GINKGO_FOCUS
+          value: "\\[PR-Informing\\]"
+        - name: GINKGO_SKIP
+          value: "\\[IPv6\\]"
+        # we need privileged mode in order to do docker in docker
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 7300m
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.1
+      testgrid-tab-name: capi-pr-e2e-informing-release-1-1
+  - name: pull-cluster-api-e2e-informing-ipv6-release-1-1
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    optional: true
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.0$
+    path_alias: sigs.k8s.io/cluster-api
+    run_if_changed: '^((api|bootstrap|cmd|config|controllers|controlplane|errors|exp|feature|hack|internal|scripts|test|third_party|util|webhooks|version)/|main\.go|go\.mod|go\.sum|Dockerfile|Makefile)'
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220121-bb77a9cb93-1.23
+        args:
+          - runner.sh
+          - "./scripts/ci-e2e.sh"
+        env:
+          - name: GINKGO_FOCUS
+            value: "\\[IPv6\\] \\[PR-Informing\\]"
+          - name: IP_FAMILY
+            value: "IPv6"
+        # we need privileged mode in order to do docker in docker
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 7300m
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-1.1
+      testgrid-tab-name: capi-pr-e2e-informing-ipv6-release-1-1
+  - name: pull-cluster-api-e2e-full-release-1-1
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    decorate: true
+    optional: true
+    always_run: false
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.0$
+    path_alias: sigs.k8s.io/cluster-api
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220121-bb77a9cb93-1.23
+        args:
+          - runner.sh
+          - "./scripts/ci-e2e.sh"
+        env:
+          # Since the PR-Blocking tests are run as part of the cluster-api-test-main job
```

```suggestion
          # Since the PR-Blocking tests are run as part of the cluster-api-test job
```
nit, let's just drop the suffix